app/views/donations/_alternatives.html.haml

```diff
@@ -2,7 +2,7 @@
   %h2.text-h5.mb-4 Looking for another payment provider?
   %p.text-p-base.mb-8 We currently support Paypal and GitHub Sponsors as alternative payment providers. Please contact #{link_to 'Loretta', 'mailto:loretta@exercism.org'} if there is another provider you feel we should add.
   .flex.flex-col.items-start.gap-12
-    = link_to "https://www.paypal.com/donate/?hosted_button_id=KDRW2GL6FVVFS", class: "btn-m btn-secondary" do
+    = link_to "#{Exercism.config.paypal_url}/donate/?hosted_button_id=#{Exercism.secrets.paypal_donation_hosted_button_id}&custom=#{current_user&.email}", class: "btn-m btn-secondary" do
```

Comment:
This adds the user in the custom field for the regular donation button. I've chosen the user's email instead of their ID/uuid to not expose that information. Let me know if you'd like this to be changed.